src/actions/index.js

```diff
@@ -555,6 +575,20 @@ function updateOutput(
                 })
             );
 
+            // Fire custom request_post hook if any
+            if (hooks.request_post !== null) {
+                if (type(hooks.request_post) === 'Function') {
+                    hooks.request_post(payload, data.response);
+                } else {
+                    /* eslint-disable no-console */
+                    // Throwing an Error or TypeError etc here will cause an infinite loop for some reason
+                    console.error(
+                        'The request_post hook provided was not of type function, preventing Dash from firing it. Please make sure the request_post hook is a function'
+                    );
```

Comment:
Nonblocking, but probably would have been better to declare that these need to be functions in somebody's `propTypes` and avoid addressing this possibility explicitly.